subsys/bootloader/Kconfig

```diff
@@ -170,7 +170,7 @@ config PM_PARTITION_SIZE_B0_IMAGE
     # The final default should be for devices without OTP, as the provision
     # image will add 0x1000 to the partition size. The following value
     # accommodates that addition to all fit within 16kB
-	default 0x2e00
+	default 0x3000
```

Comment:
This commit should be squashed in here:
0899f85d792082ddac0b83f956d34762f243bfaa

Please avoid changing the same line in multiple commit in the same PR, as it make it harder to review the PR, commit-by-commit.

Reply:
Sorry about that. Planned on cleaning up the commits after approval, but I'll just force push the rebased change(s) in the future to keep the commit history clean 👍